memory/ndctl.py

```diff
@@ -200,12 +200,6 @@ def setUp(self):
         self.reflink = '-m reflink=0'
         self.smm = SoftwareManager()
         if self.package == 'upstream':
-            # Skipping this for now, due to non-availibility of
-            # dependent packages.
-            if self.dist.name == 'SuSE' and int(self.dist.version) <= 15:
```

Comment:
@rohan-r-deshpande have we tried this on SLES12 or older SLES15 releases?

Reply:
@PraveenPenguin No we haven't

Reply:
Done